molo/profiles/templates/profiles/change_password.html

```diff
@@ -21,22 +21,22 @@
     </ul>
   {% endif %}
 
-   <div>
-      <label class="label-wide">{% trans "Old Password:" %}</label>
+   <div class="profile__fieldset">
+      <label class="profile__label">{% trans "Old Password" %}</label>
```

Comment:
I think the `label` tag is fine to style, the class doesn't add any additional information.